DTS-STN/Service-Canada-Labs
# Signup button font fix

# Description

Fixing font on Submit button to match design

## Acceptance Criteria

Get the acceptance criteria for the story and/or task you are working on and put it all here.

## Test Instructions

1. yarn dev
2. Go to signup form page
3. See Submit button using Lato font, unbolded

## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
